kubernetes/samples/scripts/domain-lifecycle/startServer.sh

```diff
@@ -0,0 +1,252 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+# This script starts a WebLogic managed server in a domain. 
```

Comment:
Question for @rjeberhard 

Anil and I went back on forth a bit about documenting the exact algorithm for server start/stop.  In the end, we came to the (not so firm) conclusion that the exact algorithm details are a bit overwhelming and unneeded for the README and 'usage' output, and that it was still minimally necessary include them as a code comment in each script (for anyone that might need to enhance/fix the scripts in the future).

Does this sound fine to you?

Reply:
Yes, that sounds fine. I agree that they are likely too complex for an end-user to have to understand all of the "rules" that we laid out last week. After the two of you have been iterating on this script, do you know think that the rules are more complicated than what we had previously listed?

Reply:
> do you now think that the rules are more complicated than what we had previously listed?

no

Reply:
thanks ryan - marking resolved